GitLfs.Core/File/LfsFileManager.cs

```diff
@@ -100,6 +100,23 @@ public string SaveFile(string repositoryName, ObjectId objectId, FileLocation lo
             }
         }
 
+        /// <inheritdoc />
+        public Stream SaveFile(out string fileName, string repositoryName, ObjectId objectId, FileLocation location, Stream contents, string suffix = null)
+        {
+            Tuple<string, string> temporaryPath = this.GetDirectoriesAndFileNames(repositoryName, objectId, location, suffix);
```

Comment:
Use C# 7 named tupes, you can do 

var (directory, fileName) = this.GetDirectoriesAndFileNames();